src/main/java/com/zemnov/salon/service/UserService.java

```diff
@@ -1,19 +1,68 @@
 package com.zemnov.salon.service;
 
+import com.zemnov.salon.dto.UserEditRequestDto;
+import com.zemnov.salon.model.Contact;
+import com.zemnov.salon.model.Role;
+import com.zemnov.salon.model.User;
+import com.zemnov.salon.repository.ContactRepo;
 import com.zemnov.salon.repository.UserRepo;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 import org.springframework.stereotype.Service;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 @Service
 public class UserService implements UserDetailsService {
     @Autowired
     private UserRepo userRepo;
+    @Autowired
+    private ContactRepo contactRepo;
 
     @Override
     public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
         return userRepo.findByUsername(username);
     }
+
+    public List<User> findAllUsers(){
+        return userRepo.findAll();
+    }
+
+    public void deleteUserById(User user){
+        contactRepo.deleteById(user.getId()+1);
```

Comment:
it looks strange), i mean ..getId() + 1. Why do we need that ?

Reply:
rude deleting of contact object from database according to contact generates instatly after generating of user. I had problems with cascade generating of related tables